localstack/services/secretsmanager/provider.py

```diff
@@ -6,22 +6,10 @@
 import time
 from typing import Final, Optional, Union
 
+import moto.secretsmanager.exceptions as MotoException
```

Comment:
Please don't deviate from Python naming conventions:

https://peps.python.org/pep-0008/#package-and-module-names

Modules must use lowercase (with underscores for separators)

Reply:
Updated. thanks for the suggestion.